new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -137,4 +232,162 @@ private static void writeToFile(Object obj, String resultDir, String fileName) t
+        .spec(new V1PersistentVolumeSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(namespace + "-weblogic-domain-storage-class")
+            .putCapacityItem("storage", Quantity.fromString("2Gi"))
+            .persistentVolumeReclaimPolicy("Recycle")
+            .hostPath(new V1HostPathVolumeSource().path(hostPath)))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvName)
+            .build());
+    TestActions.createPersistentVolume(v1pv);
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .volumeName(pvName)
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(namespace + "-weblogic-domain-storage-class")
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("2Gi"))))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvcName)
+            .withNamespace(namespace)
+            .build());
+    TestActions.createPersistentVolumeClaim(v1pvc);
+
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pv to be bound, "
+                + "(elapsed time {0} , remaining time {1}",
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(isPersistentVolumeInState(pvName, "Bound"));
+
+    final String podName = "pv-pod-" + namespace;
+    V1Pod podBody = new V1Pod()
+        .spec(new V1PodSpec()
+            .containers(Arrays.asList(
+                new V1Container()
+                    .name("pv-container")
+                    .image("nginx")
+                    .imagePullPolicy("IfNotPresent")
+                    .volumeMounts(Arrays.asList(
+                        new V1VolumeMount()
+                            .name(pvName)
+                            .mountPath("/shared")))))
+            .volumes(Arrays.asList(
+                new V1Volume()
+                    .name(pvName)
+                    .persistentVolumeClaim(
+                        new V1PersistentVolumeClaimVolumeSource()
+                            .claimName(pvcName)))))
+        .metadata(new V1ObjectMeta().name(podName))
+        .apiVersion("v1")
+        .kind("Pod");
+    V1Pod pvPod = Kubernetes.createPod(namespace, podBody);
+
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for {0} to be ready in namespace {1}, "
+                + "(elapsed time {2} , remaining time {3}",
+                podName,
+                namespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(podReady(podName, null, namespace));
+    return pvPod;
+  }
+
+  /**
+   * Delete the temporary pv pod.
+   * @param namespace name
```

Comment:
what?

Reply:
modified the description, the namespace is where the one-off pod running.